spec/numberformat.html

```diff
@@ -1594,11 +1696,17 @@ <h1>ToIntlMathematicalValue ( _value_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-getunsignedroundingmode" aoid="GetUnsignedRoundingMode">
-      <h1>GetUnsignedRoundingMode ( _roundingMode_, _isNegative_ )</h1>
-      <p>
-        The abstract operation GetUnsignedRoundingMode returns the rounding mode that should be applied to the absolute value of a number to produce the same result as if _roundingMode_, one of the String values in the *Identifier* column of <emu-xref href="#table-intl-rounding-modes"></emu-xref>, were applied to the signed value of the number (negative if the boolean _isNegative_ is *true*, or positive otherwise). The following steps are taken:
-      </p>
+    <emu-clause id="sec-getunsignedroundingmode" type="abstract operation">
+      <h1>
+        GetUnsignedRoundingMode (
+          _roundingMode_: a String,
+          _isNegative_: a Boolean,
+        ): a specification type from the *Unsigned Rounding Mode* column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref>
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It returns the rounding mode that should be applied to the absolute value of a number to produce the same result as if _roundingMode_, one of the String values in the *Identifier* column of <emu-xref href="#table-intl-rounding-modes"></emu-xref>, were applied to the signed value of the number (negative if the boolean _isNegative_ is *true*, or positive otherwise).</dd>
```

Comment:
```suggestion
        ): a specification type from the Unsigned Rounding Mode column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref>
      </h1>
      <dl class="header">
        <dt>description</dt>
        <dd>It returns the rounding mode that should be applied to the absolute value of a number to produce the same result as if _roundingMode_, one of the String values in the Identifier column of <emu-xref href="#table-intl-rounding-modes"></emu-xref>, were applied to the signed value of the number (negative if the boolean _isNegative_ is *true*, or positive otherwise).</dd>
```

Reply:
Fixed.